src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -125,8 +129,13 @@
 (schema/defn ^:always-validate new-domain :- ACE
```

Comment:
It's not really a domain anymore

Reply:
Is it not? I figured "domain" here referred to "endpoint", as in, add a new rule for a "domain". Making that rule RBAC based doesn't seem to break that concept.

Reply:
Domain here I think is a reference to the domain of the certname.

Reply:
@Magisus pointed out that the extensions also doesn't necessarily work with `new-domain` either, so maybe once we decide on the appropriate name we can update it then.